test/api/fromUrl.spec.js

```diff
@@ -0,0 +1,40 @@
+import fromUrl from '../../src/api/fromUrl'
+import * as factory from '../_fixtureFactory'
+
+describe('API - from url', () => {
+  it('should return token for file', async() => {
+    const data = await fromUrl(factory.imageUrl('valid'), {publicKey: factory.publicKey('demo')})
+
+    expect(data.uuid).toBeTruthy()
```

Comment:
`/from_url/` always return only `type` attribute, if `type` is `token`, response contain token, if - `file_info`, response also contain info about file, see https://uploadcare.com/docs/api_reference/upload/from_url/#return-values

So, you shoul check `type=token` and token attribute in response